### PR TITLE
[ALLUXIO-1981]Add the bootstrap-conf command snippet in Configuring-Alluxio-with-S3…

### PR DESCRIPTION
**Pull request by GrandpaXun**
Add the bootstrap-conf command snippet in Configuring-Alluxio-with-S3.md of the Chinese version according to the English version for Alluxio 1.1

https://alluxio.atlassian.net/browse/ALLUXIO-1981?filter=-4

### CODE REVIEW

File: docs/cn/Configuring-Alluxio-with-S3.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ priority: 0
 
 首先，本地要有Alluxio二进制包。你可以自己[编译Alluxio](Building-Alluxio-Master-Branch.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
 
+然后，如果你还没有进行这项配置，那么用‘bootstrap-conf’命令创建你的配置文件
+例如，如果你在本地机器上运行Alluxio，‘ALLUXIO_MASTER_HOSTNAME’应该设置为‘localhost’。
```
Comment (ifcharming) [inline docs/cn/Configuring-Alluxio-with-S3.md]: please use `bootstrap-conf`, `ALLUXIO_MASTER_HOSTNAME` and `localhost` for the Markdown syntax.

File: docs/cn/Configuring-Alluxio-with-S3.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ priority: 0
...
+
+{% include Configuring-Alluxio-with-s3/bootstrap.md %}
+
 然后，如果你还没有配置文件，由template文件创建配置文件：
```
Comment (ifcharming) [inline docs/cn/Configuring-Alluxio-with-S3.md]: shall we also translate this to something like
"或者，你也可以利用template文件创建配置文件并且手动设置配置内容。" ?

**Comment by ifcharming** [comment]
ok to test

**Comment by ifcharming** [comment]
thanks @GrandpaXun for the translation! I left a few minor comments.

**Comment by gpang** [comment]
Thanks for the contribution @GrandpaXun ! It looks like your commits are not associated with your github account: https://github.com/Alluxio/alluxio/pull/3457/commits

Here are instructions to get your commits associated with your github account: https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account

**Comment by ifcharming** [comment]
Hi @GrandpaXun , 

as @gpang mentioned, can you please close this PR, redo the commits linked to your github account and then reopen another one ?

**Comment by GrandpaXun** [comment]
OK!Please wait a minute.
 




------------------
-------------------------------------
王元洵
南京大学2013级本科生
计算机科学与技术系
wangyuanxun@smail.nju.edu.cn
+86 18976346928
-------------------------------------

 

 
 
 
------------------ 原始邮件 ------------------
发件人: "Chaomin Yu"; 
发送时间: 2016年6月4日(星期六) 中午1:22
收件人: "Alluxio/alluxio"; 
抄送: "yuanxun wang"; "Mention"; 
主题: Re: [Alluxio/alluxio] [ALLUXIO-1981]Add the bootstrap-conf command snippet in Configuring-Alluxio-with-S3… (#3457)

 


Hi @GrandpaXun , 
 
as @gpang mentioned, can you please close this PR, redo the commits linked to your github account and then reopen another one ?
 
&mdash;

**Pull request closed without merging**